### ISSUE TITLE
Implement cutoff dates

### ISSUE THREAD

**Issue by Iiridayn** [LTUE/call-for-panelists#14]
From Mike:

There are some definite cutoffs.
￼
1. They can't sign up for panels after the end of August.
2. We need presentation pitches by the end of August.
3. Books for consignment/traditional sellers must be submitted by Nov/Dec so that we can get the lists to the booksellers.
4. Anything that will go in the program book (Names, Bios, Schedules, etc) can no longer be changed after the first week of January.

In conclusion, we could close the second page of the call at the end of August, and leave the first page open until November or December, if we feel that is beneficial.
